backend/tracim_backend/lib/core/content.py

```diff
@@ -1002,7 +1002,9 @@ def _get_all_query(
         self,
         parent_id: int = None,
         content_type_slug: str = CONTENT_TYPES.Any_SLUG,
-        workspace: Workspace = None
+        workspace: Workspace = None,
+        label:str = None,
+        order_by_properties: typing.List[str] = (),
```

Comment:
Type hint is wrong. It should by typing.List[Column], doesnt it?

Reply:
Actually no, we currently pass strings to "order_by" function of sqlalchemy. 
We can also pass Column, complex sql subquery and probably others thing to this function, but as we have string in entry from endpoint and that string work for ordering, i do think about casting this to Column.

Reply:
@inkhey the part of code I looked at was something like `.order_by(Content.label)`. Do you really pass strings? If so, what is the string content?

Reply:
```
$ grep -r order_by *
backend/tracim_backend/lib/core/group.py:        return self._base_query().order_by(Group.group_id).all()
backend/tracim_backend/lib/core/workspace.py:            workspaces = self._base_query().order_by(Workspace.label).all()
backend/tracim_backend/lib/core/workspace.py:                .order_by(Workspace.label) \
backend/tracim_backend/lib/core/user.py:        return self._session.query(User).order_by(User.display_name)
backend/tracim_backend/lib/core/userworkspace.py:    # def get_all_for_user_order_by_workspace(
backend/tracim_backend/lib/core/userworkspace.py:    #         .join(UserRoleInWorkspace.workspace).order_by(Workspace.label).all()
backend/tracim_backend/lib/core/content.py:                    .order_by(ContentRevisionRO.revision_id.desc())
backend/tracim_backend/lib/core/content.py:            .order_by(
backend/tracim_backend/lib/core/content.py:                .order_by(Content.revision_id.desc()) \
backend/tracim_backend/lib/core/content.py:        resultset = resultset.order_by(desc(Content.updated))
backend/tracim_backend/lib/core/content.py:    #         .order_by(desc(Content.updated))
backend/tracim_backend/models/data.py:                             order_by="ContentRevisionRO.revision_id")
backend/tracim_backend/tests/models/test_content.py:            .order_by(ContentRevisionRO.revision_id.desc())\
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(Content.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(Content.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(Content.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(Content.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(ContentRevisionRO.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(ContentRevisionRO.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(ContentRevisionRO.revision_id.desc()) \
backend/tracim_backend/tests/library/test_webdav.py:            .order_by(ContentRevisionRO.revision_id.desc()) \
```

Reply:
You're right, in fact, i pass "Content.label" which is "'hybrid_propertyProxy" .... i just tried with passing "label" and it work also.

Reply:
Actually, the problem is that typing must help developer to understand the code. If the typing is defined as "string", the developer has no information about what to give as a parameter. I see 2 possibilities : 

- improve typing (the best solution)
- add docstring

Maybe both must be implemented.

Reply:
problem i see is that typing who should allowed here is clearly unclear.

Reply:
as soos as you fix this, tell me (or @buxx) to merge the MR